Errors building vscode-solidity due to wrong dependency version

I get the following errors building vscode-solidity:

````
node_modules/vscode-languageclient/lib/declaration.d.ts:1:75 - error TS2305: Module '"vscode"' has no exported member 'Declaration'.

1 import { Disposable, TextDocument, ProviderResult, Position as VPosition, Declaration as VDeclaration } from 'vscode';
                                                                            ~~~~~~~~~~~

node_modules/vscode-languageclient/lib/protocolConverter.d.ts:40:59 - error TS2694: Namespace '"vscode"' has no exported member 'LocationLink'.

40     asDeclarationResult(item: ls.DeclarationLink[]): code.LocationLink[];
                                                             ~~~~~~~~~~~~

node_modules/vscode-languageclient/lib/protocolConverter.d.ts:42:95 - error TS2694: Namespace '"vscode"' has no exported member 'Declaration'.

42     asDeclarationResult(item: ls.Declaration | ls.DeclarationLink[] | undefined | null): code.Declaration | undefined;
        
````

The problem is that package.json specifies vscode version 1.28 or later, but those symbols are defined in version 1.30.

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
   "version": "0.0.64",
   "publisher": "JuanBlanco",
   "engines": {
-    "vscode": "^1.28.0"
+    "vscode": "^1.30.0"
   },
   "contributors": [
     "Juan Blanco"

